plugins/animation_to_java.js

```diff
@@ -0,0 +1,84 @@
+(function () {
+  var menuButton;
+
+  Plugin.register("animation_to_java", {
+    title: "Animation to Java Converter",
+    author: "MG",
+    description:
+      "Converts Blockbench animations to Java code for the new 1.19 keyframe system",
+    icon: "fa-cube",
+    version: "1.0.0",
+    variant: "both",
+    tags: ["Minecraft: Java Edition"],
+    onload() {
+      menuButton = new Action("export_to_java", {
+        name: "Export Animations to Java",
+        description: "Exports animations to Java code",
+        icon: "fa-file-export",
+        click: function () {
+          var file = generateFile();
+          Blockbench.export({
+            type: "Text File (.txt)",
```

Comment:
No need to include txt here, that's automatically added
![image](https://user-images.githubusercontent.com/29278436/174665622-47724dc1-6965-4002-b5bd-daccab865214.png)